cmd/wego/add/cmd.go

```diff
@@ -94,6 +95,8 @@ func runCmd(cmd *cobra.Command, args []string) error {
 
 	appService := app.New(gitClient, fluxClient, kubeClient, gitProviders)
 
+	utils.SetCommmitMessageFromArgs("wego app add", params.Url, params.Path, params.Name)
```

Comment:
The plan is to set this to the be the exact command that was run. I was unware exactly how to do this and this PR needed done faster so I am pushing this to the backburner.